causalml/inference/tree/uplift.pyx

```diff
@@ -229,12 +229,13 @@ class UpliftTreeClassifier:
 
     """
     def __init__(self, control_name, max_features=None, max_depth=3, min_samples_leaf=100,
-                 min_samples_treatment=10, n_reg=100, evaluationFunction='KL',
+                 min_samples_treatment=10, n_reg=100, eval_diff=0.01, evaluationFunction='KL',
```

Comment:
nit: will it be more intuitive to understand if we use `early_stopping_eval_diff` similar like the `early_stopping_rounds` in other packages?